recipes/triqs_cthyb/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set version = "2.2.0" %}
+
+package:
+  name: triqs_cthyb
+  version: {{ version }}
+
+source:
+  url: https://github.com/TRIQS/cthyb/releases/download/{{ version }}/cthyb-{{ version }}.tar.gz
+  sha256: 89b572027f88c6b9afd176bd36023d646efba056ce90dea1af57b662fd713ad5
+
+build:
+  number: 0
+  skip: True  # [win or py>30]
+  merge_build_host: True
+
+requirements:
+  build:
+    - cmake
+    - make
+    - clangxx >=8  # [osx]
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - triqs 2.2
+    - nfft
+  run:
+    - triqs 2.2
+    - nfft
+
+test:
+  imports:
+    - triqs_cthyb 
+
+about:
+  home: https://triqs.github.io/cthyb
+  license: GPLv3
+  license_file: LICENSE.txt
```

Comment:
Please add a license_family entry. CC @conda-forge/core this would have been caught by https://github.com/conda-forge/conda-smithy/pull/1256

Reply:
Done